### DISCUSSION TITLE
Protocol Missing in wiki

### DISCUSSION THREAD

**Discussion by robthomson** [pascallanger/DIY-Multiprotocol-TX-Module#720] [General]
Noticed that on this page:

https://www.multi-module.org/basics/supported-protocols

Protocol 21 for futaba is missing on the list.

**Comment by pascallanger** [comment]
I'm maintaining this page: https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/blob/master/Protocols_Details.md

**Comment by benlye** [comment]
I'll take care of it. I'm a few protocols behind, I think.
